### PR TITLE
Simplify compilation to wasm by using wasienv

### PR DESCRIPTION
**Pull request by syrusakbary**
This PR simplifies the generation of WebAssembly/WASI files by using [wasienv](https://github.com/wasienv/wasienv).

More about wasienv here:
https://medium.com/wasmer/wasienv-wasi-development-workflow-for-humans-1811d9a50345

### CODE REVIEW

File: qjs-wasi/CMakeLists.txt [hunk 1]
```diff
@@ -19,7 +19,7 @@ add_definitions(
     -DCONFIG_VERSION="2019-07-09"
 )
 
-add_executable(qjs.wasm
+add_executable(qjs
```
Comment (syrusakbary) [inline qjs-wasi/CMakeLists.txt:22]: Note, the `qjs.wasm` will be automatically generated (and `qjs` will be just an executable wrapper to execute the wasm file with a standalone WebAssembly runtime)

**Comment by saghul** [comment]
Amazing work @syrusakbary ! This makes things soooo much easier!

I do have a minor issue when running the wrapper script:

```
./build/qjs
Traceback (most recent call last):
  File "/Users/saghul/Library/Python/2.7/bin/wasirun", line 9, in <module>
    load_entry_point('wasienv==0.3.0', 'console_scripts', 'wasirun')()
  File "/Users/saghul/Library/Python/2.7/lib/python/site-packages/wasienv/commands.py", line 46, in wasirun
    return execute(wasirun)
  File "/Users/saghul/Library/Python/2.7/lib/python/site-packages/wasienv/tools.py", line 172, in execute
    sys.exit(f(sys.argv))
TypeError: wasirun() takes no arguments (1 given)
```

**Comment by syrusakbary** [comment]
Got it!

Thanks for the report. I just fixed it with the commit: https://github.com/wasienv/wasienv/commit/8b2cc39d0ebc5190a8b2d8af4ce97c091988a7dd

Once wasienv is reinstalled, it will be automatically fixed! :)

**Comment by saghul** [comment]
Working like a charm! ❤️

**Pull request merged**